Update managed-service-provider.md (#16184)
pingcap/docs: tidb-cloud/managed-service-provider.md

---

# Managed Service Provider

> The Managed Service Partner Program is designed to create and support a close working relationship between PingCAP and our partners to benefit our common customers. 

Reviewer: ```suggestion

The Managed Service Partner Program aims to establish and foster a strong partnership between PingCAP and our partners to benefit our customers. 
```

A managed service provider (MSP) is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.

> Benefits to the managed service provider include:
> - Discounts and Incentive Program
> - Enablement training
> - Increased visibility through certification
> - Joint marketing opportunities

Reviewer: ```suggestion
Benefits of becoming a managed service provider include:

- Discounts and incentive programs
- Enablement training
- Increased visibility through certification
- Joint marketing opportunities
```

## Become an MSP of PingCAP

> If you are interested in the MSP program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll. Please prepare the following info:

Reviewer: ```suggestion
If you are interested in the MSP program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll. Please provide the following information:
```

- Company name
- Company contact email
- Company official website URL

> - Company logo (one SVG file for light mode and one SVG file for dark mode. Horizontal logo with 256 x 48 pixels is preferred)

Reviewer: ```suggestion
- Company logo (One SVG file for light mode and one SVG file for dark mode. A horizontal logo with 256 x 48 pixels is preferred)
```

We will carefully evaluate your request and get back to you soon.

## Manage daily tasks for MSP customers

Once you are approved as a PingCAP MSP, you will receive an API key for the [MSP Management API](https://docs.pingcap.com/tidbcloud/api/msp/v1beta1). 

> You can use MSP management API to manage daily tasks:

Reviewer: ```suggestion
You can use the MSP management API to manage daily tasks:

```

> - Query MSP monthly bill for a given month
> - Query credits applied to MSP 
> - Query discounts applied to MSP 
> - Query monthly bill for a specific MSP customer
> - Create a new signup URL for new MSP customer
> - List all MSP customers 
> - Retrieve MSP customer info by organization ID

Reviewer: ```suggestion
- Query the MSP monthly bill for a specific month
- Query credits applied to an MSP 
- Query discounts applied to an MSP 
- Query the monthly bill for a specific MSP customer
- Create a new signup URL for an MSP customer
- List all MSP customers 
- Retrieve MSP customer information by the customer organization ID
```

# Managed Service Provider

The Managed Service Partner Program aims to establish and foster a strong partnership between PingCAP and our partners to benefit our customers. 

A managed service provider (MSP) is a partner who resells TiDB Cloud and provides value-added services, including but not limited to TiDB Cloud organization management, billing services, and technical support.

Benefits of becoming a managed service provider include:

- Discounts and incentive programs
- Enablement training
- Increased visibility through certification
- Joint marketing opportunities

## Become an MSP of PingCAP

> If you are interested in the MSP program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll. Please provide the following information:

Reviewer: ```suggestion
If you are interested in the MSP program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll. Please provide the following information:

```
